rotkehlchen/globaldb/upgrades/v3_v4.py

```diff
@@ -252,6 +260,12 @@ def _add_optimism_contracts(cursor: 'DBCursor', eth_scan_abi_id: int, multicall_
             'MULTICALL2',
             multicall_abi_id,
             722566,
+        ), (
```

Comment:
Shouldn't this addition to the DB upgrade also be tested for in the relevant DB test?

Reply:
Yes, added